libdnf.spec

```diff
@@ -345,17 +348,21 @@ Package management service with a DBus interface
 # ========== microdnf ==========
 
 %if %{with microdnf}
-%package -n microdnf
-Summary:        Package management service with a DBus interface
+%package -n microdnf5
+Summary:        Command-line package manager
 License:        GPLv2+
-Requires:       libdnf%{?_isa} = %{version}-%{release}
+Provides:       microdnf
```

Comment:
This needs to be a versioned capability: `Provides: microdnf = %{version}-%{release}`

Reply:
Good point, let me change it.